packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -16,29 +26,35 @@ interface SoqlEditorEvent {
 export enum MessageType {
   ACTIVATED = 'activated',
   QUERY = 'query',
+  SOBJECT_METADATA_REQUEST = 'sobject_metadata_request',
+  SOBJECT_METADATA_RESPONSE = 'sobject_metadata_response',
+  SOBJECTS_REQUEST = 'sobjects_request',
+  SOBJECTS_RESPONSE = 'sobjects_response',
   UPDATE = 'update'
 }
 
 export class SOQLEditorInstance {
```

Comment:
Changed privacy scope of most members from `private` to `protected` for visibility to tests via subclassing.

Reply:
Not for this PR:  How about moving `MessageType` and `SoqlEditorEvent` to `soql-tooling` ? We need them on that side, and vscode-soql depends on soql-tooling.